### PR TITLE
use nightlies.pl.com

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: fb-install-puppet.bats [hunk 1]
```diff
@@ -66,17 +66,17 @@ load os_helper
 @test "setup Puppet Labs nightly repos" {
   [ x${PUPPET_REPO} = xnightly ] || skip "PUPPET_REPO is not set to nightly"
   tSetOSVersion
+  tPackageExists curl || tPackageInstall curl
   if tIsFedora; then
-    tPackageExists puppet5-nightly-release || \
-      rpm -ivh http://yum.puppetlabs.com/puppet5-nightly/puppet5-nightly-release-fedora-${OS_VERSION}.noarch.rpm
+    curl -o /etc/yum.repos.d/puppet-agent.repo http://nightlies.puppetlabs.com/puppet-agent-latest/repo_configs/rpm/pl-puppet-agent-latest-fedora-${OS_VERSION}-x86_64.repo
```
Comment (ekohl) [inline fb-install-puppet.bats]: How about using ```wget -O``` like we do in the rest of the file?

Comment (mmoll) [reply fb-install-puppet.bats]: good point, changed.

File: fb-install-puppet.bats [hunk 2]
```diff
@@ -67,16 +67,15 @@ load os_helper
...
-    tPackageExists puppet5-nightly-release || \
-      rpm -ivh http://yum.puppetlabs.com/puppet5-nightly/puppet5-nightly-release-el-${OS_VERSION}.noarch.rpm
+    wget -O /etc/yum.repos.d/puppet-agent.repo http://nightlies.puppetlabs.com/puppet-agent-latest/repo_configs/rpm/pl-puppet-agent-latest-el-${OS_VERSION}-x86_64.repo
+    wget -O /etc/yum.repos.d/puppetserver.repo http://nightlies.puppetlabs.com/puppetserver-latest/repo_configs/rpm/pl-puppetserver-latest-el-${OS_VERSION}-x86_64.repo
   elif tIsDebianCompatible; then
-    if ! tPackageExists puppet5-nightly-release; then
-      wget http://apt.puppetlabs.com/puppet5-nightly-release-${OS_RELEASE}.deb
-      dpkg -i puppet5-nightly-release-${OS_RELEASE}.deb
-    fi
+    wget -qO- http://nightlies.puppetlabs.com/07BB6C57 | apt-key add -
```
Comment (ekohl) [inline fb-install-puppet.bats]: Might be better to retrieve this over https

Comment (mmoll) [reply fb-install-puppet.bats]: Using https for all nighlty URLs now, added `tPackageUpgrade ca-certificates` for Debian to make sure this works.

**Pull request merged**